build.sbt

```diff
@@ -145,22 +146,25 @@ lazy val microsite = project.module
         |<p>&copy; 2018 <a href="https://github.com/scalaz/scalaz-zio">ZIO Maintainers</a></p>
         |""".stripMargin
     ),
-    micrositeName := "ZIO",
-    micrositeDescription := "ZIO",
+  /*  micrositeCssDirectory := "/microsite/resources/microsite/css/main.css",*/
+
+    micrositeName := "",
+    micrositeDescription := "Type-safe, compositional concurrency for Scala",
     micrositeAuthor := "ZIO contributors",
     micrositeOrganizationHomepage := "https://github.com/scalaz/scalaz-zio",
     micrositeGitterChannelUrl := "scalaz/scalaz-zio",
     micrositeGitHostingUrl := "https://github.com/scalaz/scalaz-zio",
     micrositeGithubOwner := "scalaz",
     micrositeGithubRepo := "scalaz-zio",
     micrositeFavicons := Seq(microsites.MicrositeFavicon("favicon.png", "512x512")),
+    micrositeHighlightTheme := "atom-one-light",
     micrositeDocumentationUrl := s"https://javadoc.io/doc/org.scalaz/scalaz-zio_2.12/${(version in Compile).value}",
     micrositeDocumentationLabelDescription := "Scaladoc",
     micrositeBaseUrl := "/scalaz-zio",
     micrositePalette := Map(
```

Comment:
Are these colors being used, or are they overridden in the CSS file?

Reply:
overriden in css file

Reply:
We don't need this line

Reply:
@regiskuckaertz , @rssh can you review it again?